packages/vscode-extension/package.json

```diff
@@ -1091,6 +1092,7 @@
     "@rollup/rollup-linux-x64-gnu": "4.40.0"
   },
   "dependencies": {
-    "@tanstack/react-virtual": "^3.13.12"
+    "@tanstack/react-virtual": "^3.13.12",
+    "react-shiki": "^0.8.0"
```

Comment:
We generally add all of the dependencies as `devDependencies`. Would be good to move these there as well.